core/src/rpc_subscriptions.rs

```diff
@@ -11,17 +11,47 @@ use solana_sdk::{
 };
 use solana_vote_program::vote_state::MAX_LOCKOUT_HISTORY;
 use std::collections::HashMap;
+use std::sync::atomic::{AtomicBool, Ordering};
+use std::sync::mpsc::{Receiver, SyncSender, TryRecvError, TrySendError};
 use std::sync::{Arc, RwLock};
+use std::thread::{Builder, JoinHandle};
+use std::time::Duration;
+
+const NOTIFICATION_QUEUE_SIZE: usize = 1_000;
+const RECV_DELAY_MILLIS: u64 = 100;
```

Comment:
Prefer real words please:
```suggestion
const RECEIVE_DELAY_MILLIS: u64 = 100;
```